### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by Igor-Pysak**
- [DEMO LINK](https://Igor-Pysak.github.io/layout_search-bar-airbnb/)
- [TEST REPORT LINK](https://Igor-Pysak.github.io/layout_search-bar-airbnb/report/html_report/)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+  font-size: 14px;
+  font-weight: 300;
+  color: var(--color-placeholder);
+}
+
+.search-form__img {
+  position: absolute;
+  width: 19px;
+  height: 19px;
+  padding: 25px 0 0 26px;
```
Comment (Viktor-Kost) [inline src/style.css:83]: use top/left rules to position absoluted icon
```suggestion
```

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+}
+
+.search-form__img--small {
+  width: 11px;
+  height: 11px;
+  padding: 15px 0 0 13px;
```
Comment (Viktor-Kost) [inline src/style.css:89]: the same with top/left

**Review by Viktor-Kost — CHANGES_REQUESTED** [review]
Great job!
Some moments can be improved

**Review by volodymyr-soltys97 — APPROVED** [review]
Good job 👍

**Pull request closed without merging**